### PR TITLE
Udevd more configuration options

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM.

File: src/udev/udev.h [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #include "macro.h"
 #include "strv.h"
 #include "util.h"
+#include "udev-util.h"
```
Comment (poettering) [inline src/udev/udev.h:21]: not ordered alphabetically, but doesn't matter

**Comment by keszybz** [comment]
```
autopkgtest [15:11:05]: test logind: [-----------------------
 * try-restarting systemd-logind
 * daemon is started
 * loginctl succeeds
 * XDG_SESSION_ID=
1:name=systemd:/user.slice/user-1000.slice/session-1.scope
0::/user.slice/user-1000.slice/session-1.scope
 * process is in session cgroup
-- Logs begin at Thu 2018-11-08 03:16:33 UTC. --
Nov 13 15:00:57 autopkgtest systemd-logind[387]: New session 1 of user ubuntu.
Nov 13 15:01:19 autopkgtest systemd[1]: /lib/systemd/system/systemd-logind.service:46: Unknown system call group, ignoring: @system-service
Nov 13 15:11:05 autopkgtest systemd[1]: Stopping Login Service...
Nov 13 15:11:05 autopkgtest systemd[1]: Stopped Login Service.
Nov 13 15:11:05 autopkgtest systemd[1]: Starting Login Service...
Nov 13 15:11:05 autopkgtest systemd-logind[9852]: Watching system buttons on /dev/input/event0 (Power Button)
Nov 13 15:11:05 autopkgtest systemd-logind[9852]: Watching system buttons on /dev/input/event1 (AT Translated Set 2 keyboard)
Nov 13 15:11:05 autopkgtest systemd-logind[9852]: New seat seat0.
Nov 13 15:11:05 autopkgtest systemd[1]: Started Login Service.
Nov 13 15:11:05 autopkgtest systemd-logind[9852]: New session 1 of user ubuntu.
Fake Lid Switch: /dev/input/event4
Nov 13 15:11:07 autopkgtest systemd-logind[9852]: Watching system buttons on /dev/input/event4 (Fake Lid Switch)
   waiting for suspend, 30s remaining...
   waiting for suspend, 25s remaining...
   waiting for suspend, 20s remaining...
   waiting for suspend, 15s remaining...
   waiting for suspend, 10s remaining...
   waiting for suspend, 5s remaining...
   waiting for suspend, 0s remaining...
closing lid did not cause suspend
autopkgtest [15:11:38]: test logind: -----------------------]
```
logind fails. This *could* be related, but the test runners seem to be starved for resources, so possibly it's just a timeout. Let's monitor if the same failure appears once the runners are recovered.

**Comment by evverx** [comment]
In https://github.com/systemd/systemd/pull/10766, I can see the same failure of the logind test so it's very unlikely that this PR has anything to do with. It's also very unlikely that the runners are recovered as long as the test keeps failing.

By the way, I opened https://github.com/systemd/systemd/issues/10774.

**Comment by keszybz** [comment]
@evverx thanks for the info. I'll set the tag.

**Pull request merged**